src/changeAdminStatus.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function changeAdminStatus(userIDs, makeAdmin, threadID, callback) {
+    if (
+      !callback &&
+      (utils.getType(makeAdmin) === "AsyncFunction" ||
+        utils.getType(makeAdmin) === "AsyncFunction" ||
+        utils.getType(threadID) === "Function" ||
+        utils.getType(threadID) === "AsyncFunction")
+    ) {
+      throw { error: "please pass makeAdmin and threadID as the second/third arguments." };
```

Comment:
If these are required fields, we can probably just check their types rather than checking if they're not a callback?

Reply:
Yep, another holdover from the basis file I used.